readme.md

```diff
@@ -77,6 +77,8 @@ Directus only requires Node.js and supports most operating systems and SQL datab
 - Windows 10
 - Docker ([DockerHub](https://hub.docker.com/r/directus/directus) +
   [Dockerfile](https://github.com/directus/directus/blob/main/docker/Dockerfile))
+- Cloudron ([Package Source](https://git.cloudron.io/cloudron/directus-app) +
+  [Cloudron Store link](https://www.cloudron.io/store/io.directus9.cloudronapp.html))
```

Comment:
Perhaps we can move this line (particular store link since package source is already added) to the installation guide instead of under `Supported OS` here? 🤔 Though I do understand Docker would arguably be the exception entry here in the first place.

The `One-Click Deployment` table (starting from line 47) should be a perfect fit as well! In accordance to the addition in `installation.md` in the current PR 👍

Reply:
I missed the 1-click deployment section entirely. I will move it there.